app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -92,14 +92,16 @@ export const args: Arguments = await yargs(process.argv.slice(2))
 // ======================================
 
 /** Based on the given arguments, creates a configuration for the Electron Builder. */
-export function createElectronBuilderConfig(passedArgs: Arguments): electronBuilder.Configuration {
+export async function createElectronBuilderConfig(
```

Comment:
does this need to be async?

Reply:
Not anymore, this is remnant from my previous iterations.